sys/shell/commands/sc_gnrc_rpl.c

```diff
@@ -292,8 +292,9 @@ int _gnrc_rpl_dodag_show(void)
                 gnrc_rpl_instances[i].mop, gnrc_rpl_instances[i].of->ocp,
                 gnrc_rpl_instances[i].min_hop_rank_inc, gnrc_rpl_instances[i].max_rank_inc);
 
-        tc = (((uint64_t) dodag->trickle.msg_timer.long_target << 32)
-                | dodag->trickle.msg_timer.target) - xnow;
+        tc = _xtimer_usec_from_ticks64(((uint64_t) dodag->trickle.msg_timer.long_offset << 32)
+                                        | dodag->trickle.msg_timer.offset)
+             - (xnow - _xtimer_usec_from_ticks64(dodag->trickle.msg_timer.start_time));
```

Comment:
The same concurrency problem here (already was there before and is probably not that critical). 
But from looking at this, maybe a simple (but also irq-guarded) way to get the time left on a timer would be a nice thing (?)

Reply:
> While the concurrency problem was there before we should still fix it now

same here